scripts/codegen

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+source $(dirname $0)/lib/debug_functions
+
+K8S_IO_DIR="${GOPATH:-~/go}/src/k8s.io"
+CODEGEN_SCRIPT_DIR="${K8S_IO_DIR}/code-generator"
+CODEGEN_SCRIPT="${CODEGEN_SCRIPT_DIR}/generate-groups.sh"
+CODEGEN_RELEASE_TAG=kubernetes-1.14.1
+export GO111MODULE=off
+mkdir -p $K8S_IO_DIR
+
+if [ ! -f "$CODEGEN_SCRIPT" ]; then
+    echo "$CODEGEN_SCRIPT does not exist - downloading..."
+    cd $K8S_IO_DIR
+    git clone --branch $CODEGEN_RELEASE_TAG https://github.com/kubernetes/code-generator
+    cd -
+    echo "Successsfully checked out release tag $CODEGEN_RELEASE_TAG"
+fi
+
+$CODEGEN_SCRIPT all github.com/submariner-io/lighthouse/pkg/client github.com/submariner-io/lighthouse/pkg/apis lighthouse.submariner.io:v2alpha1
```

Comment:
```suggestion
$CODEGEN_SCRIPT all github.com/submariner-io/lighthouse/pkg/client github.com/submariner-io/lighthouse/pkg/apis lighthouse.submariner.io:v1,v2alpha1
```

This will generate client code for both v1 and v2alpha1, including informers.